/* { dg-do run } */

defun foo = int:
  {
   try
     raise Exception { code = 0} ;
   catch
   {
    print ("got an exception\n");
   }
   return 666;
  }

/* { dg-command {  foo } } */
/* { dg-output "got an exception" } */
